src/Utilities/for_testing/NativeInitializerPrint.hpp

```diff
@@ -95,7 +95,10 @@ inline std::ostream& operator<<(std::ostream& out, const NativePrint<Vector<T>>&
   out << "{ ";
   auto vec = np_vec.get_obj();
   for (T& t : vec)
-    out << std::setprecision(10) << t << ", ";
+    if constexpr (IsComplex_t<T>::value)
+      out << std::setprecision(10) << '{' << t.real() << ", " << t.imag() << "}, ";
+    else
+      out << std::setprecision(10) << t << ", ";
```

Comment:
What is your motivation of changing the style of std::complex printout?
I tend to prefer what the standard library does.
If we change the style in this template specialization, should we all the cases in this file?

Reply:
It needs to be correct for the initializer of a Vector of std::complex.  That is the entire point of this output adapter is to output in a format that can be directly pasted into a test program for expected output.  That doesn't work with the default operator<< for std::complex.

Reply:
Basically it is a special need. Q: does the extra ‘, ‘ causes problem or you always remove it by hand.